.github/workflows/ci_release.yml

```diff
@@ -76,7 +76,7 @@ jobs:
   tests:
     uses: ./.github/workflows/tests.yml
     with:
-      ANSYS_VERSION: ${{ github.event.inputs.ansys_version || '251' }}
+      ANSYS_VERSION: ${{ github.event.inputs.ansys_version || '252' }}
```

Comment:
@guptadivya1501 We will also need a retro_251 test just like for retro_241. We actually also need a retro_242